vizro-core/docs/pages/user-guides/graph.md

```diff
@@ -15,6 +15,9 @@ To add a [`Graph`][vizro.models.Graph] to your page, do the following:
     In order to use the [`plotly.express`](https://plotly.com/python/plotly-express/) chart in a Vizro dashboard, you need to import it as `import vizro.plotly.express as px`.
     This leaves any of the [`plotly.express`](https://plotly.com/python/plotly-express/) functionality untouched, but allows _direct insertion_ into the [`Graph`][vizro.models.Graph] model _as is_.
 
+    Note also that the `plotly.express` chart needs to have a `data_frame` argument. This currently holds for all but the
+    [`imshow` chart](https://plotly.com/python/imshow/). In case you require the `imshow` chart, please refer to our [guide on custom charts](custom-charts.md).
```

Comment:
I am not sure I understand, I clicked through all available `plotly.express` charts, and the `px.imshow` was the only one I discovered that didn't have a `data_frame` argument.

I checked specifically so we could be sure for this comment - as it's quite nice to state that all charts are possible.

Reply:
Ah didn't know you've checked all px.charts - that's good to know! fine to keep it as you have it then. 

I think keeping it more generic would have the advantage that he we wouldn't have to update this in case they do add more charts into their library that don't have a `data_frame` argument. Might also be worth linking to this: https://plotly.com/python-api-reference/plotly.express.html

Gives a good overview as well which plotly.express figures could be used in general?

Reply:
Ah also good link, in this case would have been more useful haha :). I used this one here: https://plotly.com/python/plotly-express/, which is also linked in the docs. I think in that case probably that one is more useful because it is more descriptive and groups the charts a little

I changed the text a little so it would always hold, but still mention the `imshow` 👍